.changeset/eight-oranges-wave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend-module-microsoft-provider': patch
```

Comment:
Gotta update this one now 😅

Reply:
🤦 - I remembered to update the changeset text, but not the package name.